runtime/doc/eval.txt

```diff
@@ -5405,13 +5405,25 @@ server2client( {clientid}, {string})			*server2client()*
 			:echo server2client(expand("<client>"), "HELLO")
 <
 serverlist()					*serverlist()*
-		Return a list of available server names, one per line.
-		When there are no servers or the information is not available
-		an empty string is returned.  See also |clientserver|.
-		{only available when compiled with the |+clientserver| feature}
+		Return a list of available server names in a list.
```

Comment:
:+1: -- I'm not entirely sure which is more appropriate, though. Most vim docs use commanding language like "return the value", while neovim style is more descriptive "returns the value".